### PR TITLE
refactor: remove labs package

### PR DESCRIPTION
**Pull request by alexeagle**
See #2892

BREAKING CHANGE:
We no longer supply the @bazel/labs package.
The only rule in it, ts_proto_library, has been neglected and is tied to the
deprecated concatjs package.
Also, there is great fragmentation in proto/gRPC ecosystem under Bazel,
and having one obscure option in rules_nodejs makes this worse.

### CODE REVIEW

**Review by mattem — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This repository is maintained by volunteers in the Bazel community. Neither Goog
 We follow semantic versioning. Patch releases have bugfixes, minor releases have new features. Only major releases (1.x, 2.x) have breaking changes. We support [LTS releases](https://blog.bazel.build/2020/11/10/long-term-support-release.html) of Bazel (starting at 4.x), see `SUPPORTED_BAZEL_VERSIONS` in our `/index.bzl` for the list we test against.
 
 We strive to give you an easy upgrade path when we do introduce a breaking change by documenting a migration path.
-If you use code from an `/internal` path, or the labs package, these are not subject to our support policy and may have breaking changes or removals with no warning or migration path.
+If you use code from an `/internal` path, these are not subject to our support policy and may have breaking changes or removals with no warning or migration path.
```
Comment (mattem) [inline README.md:18]: nit
```suggestion
Using code from an `/internal` path is not subject to our support policy and may have breaking changes or removals with no warning or migration path.
```

**Comment by alexeagle** [comment]
oops I didn't see the suggestions on mobile when I merged... will include in a later one

**Comment by reddaly** [comment]
@alexeagle  Do you have recommendations about what to use for proto/grpc-web support?

**Comment by alexeagle** [comment]
https://bazel-contrib.github.io/SIG-rules-authors/proto-grpc.html is probably the place to start. I don't think there is one canonical answer.

**Pull request merged**